cores/arduino/gd32/uart.c

```diff
@@ -33,7 +33,7 @@ OF SUCH DAMAGE.
 #if defined(USART_DATA)
 #define GD32_USART_TX_DATA USART_DATA
 #define GD32_USART_RX_DATA USART_DATA
-#define GD32_USART_STAT    USART_STAT0
+#define GD32_USART_STAT    USART_STAT
```

Comment:
Does this really want to change for all devices? It feels like instead we might want to define some compat macros that are ifdefed to the F10x so we don't need to make this change inside the arduino core?

Reply:
fixed